conntrack_linux.go

```diff
@@ -139,6 +139,14 @@ func (h *Handle) ConntrackUpdate(table ConntrackTableType, family InetFamily, fl
 	return err
 }
 
+// ConntrackDeleteFilter deletes entries on the specified table on the base of the filter using the netlink handle passed
+// conntrack -D [table] parameters         Delete conntrack or expectation
+//
+// Deprecated: use [Handle.ConntrackDeleteFilters] instead.
```

Comment:
why is this being marked as deprecated, when the top-level `ConntrackDeleteFilter` function is not?

Reply:
My thinking here was that the new one effectively replaces the existing one. So deprecation as indication for users to move over.


I'm fine removing though (honestly I think the change in signature probably would've been fine without a rename, but I guess it could break someone trying to match against an interface)

Reply:
My point was that if this method is marked as deprecated, then `ConntrackDeleteFilter` should also be marked as deprecated, or it's inconsistent

https://github.com/vishvananda/netlink/blob/65a253d3751cff6da8274035e4548974a874f477/conntrack_linux.go#L72

Reply:
Ah! Sorry, misinterpreted your comment 😅 

Yes, that makes sense; I pushed, and deprecated that one as well


While working on that, I also found that there c96b03b4befaf5cc418386ad9eaf1a5d15101b1e did not add stubs for non-supported platforms, so I added those as well.